lib/client/js/state/Store.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @class Store
+ */
+var Store = (function() {
+    /*
+     * @alias Store
+     * @constructor
+     * @param {Object} initialProps - A custom props object
+     * @returns {Store}
+     */
+    var Store = function(initialProps) {
```

Comment:
Question: A lot of the other classes that we've made so far have their constructor signatures as follows:
`function Store(initialProps) {}`
Is there any particular reason for having the signature be like `var Store = function(initialProps) {}`?